Cargo.toml

```diff
@@ -184,7 +184,8 @@ tokio = { version = "1", features = ['full'] }
 tokio-stream = { version = "0.1", features = ["fs", "io-util"] }
 tokio-util = { version = "0.7", features = ["compat", "io-util"] }
 toml = "0.8"
-tower = { version = "0.5", features = ["full"] }
+tower = { version = "0.4", features = ["util"] }
```

Comment:
downgrade `tower` to align with `jsonrpsee`

Reply:
@hanabi1224 Should we put a comment here so that people avoid updating (and a similar one in the `jsonrpsee` so that people update it when needed).

Reply:
@LesnyRumcajs Now It does not compile if tower and jsonrpsee versions don't match

Reply:
@LesnyRumcajs Now It does not compile if tower and jsonrpsee versions don't match